packages/core/data-transfer/src/strapi/providers/local-destination/strategies/restore/configuration.ts

```diff
@@ -21,30 +24,30 @@ export const restoreConfigs = async (strapi: Strapi.Strapi, config: IConfigurati
   }
 
   if (config.type === 'webhook') {
-    return restoreWebhooks(strapi, config.value);
+    return restoreWebhooks(strapi, config.value as { value: unknown });
   }
 };
 
-export const createConfigurationWriteStream = async (strapi: Strapi.Strapi) => {
+export const createConfigurationWriteStream = async (strapi: Strapi.Strapi, transaction: any) => {
   return new Writable({
     objectMode: true,
     async write<T extends { id: number }>(
       config: IConfiguration<T>,
       _encoding: BufferEncoding,
       callback: (error?: Error | null) => void
     ) {
-      try {
-        await restoreConfigs(strapi, config);
-      } catch (error) {
-        return callback(
-          new Error(
-            `Failed to import ${chalk.yellowBright(config.type)} (${chalk.greenBright(
-              config.value.id
-            )}`
-          )
-        );
-      }
-      callback();
+      return transaction(async () => {
+        try {
+          await restoreConfigs(strapi, config);
+        } catch (error) {
+          return callback(
+            new Error(
+              `Failed to import ${chalk.yellowBright(config.type)} (${chalk.greenBright(error)}`
```

Comment:
We probably don't want to include colors at this level, but we can worry about that once the diagnostics are merged and we can pass this data to a higher level to let the logger deal with it.

Reply:
I am didn't add this it always existed, I just changed it from id to error, I wanted to see the error but indeed